lib/debootstrap.sh

```diff
@@ -616,6 +616,21 @@ prepare_partitions()
         fi
 	fi
 
+	if [[ $SRC_EXTLINUX == yes ]]; then
```

Comment:
This part should be moved here:
https://github.com/armbian/build/blob/master/lib/distributions.sh#L169-L186